lib/pages/sign_up.dart

```diff
@@ -106,6 +115,13 @@ class _SignUpState extends State<SignUp> {
       body: SafeArea(
         child: BlocBuilder<RegistrationCubit, RegistrationState>(
           bloc: Get.find<RegistrationCubit>(),
+          buildWhen: (_, currentState) =>
```

Comment:
Why list all the states? Just remove the buildWhen...

Reply:
Why this one has not been fixed?

Reply:
Man, it's fixed it's outdated code